src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action=" "
```

Comment:
The `action` attribute of the `form` element is currently set to an empty string. This means that the form will not submit data to any server endpoint. If this is not intentional, you should specify a valid URL or endpoint where the form data should be sent.